### PR TITLE
[Outlook] (mobile web support) Update modern Outlook on web add-in support information for mobile phones

### PR DESCRIPTION
**Pull request by samantharamon**
Clarify that add-ins don't work on modern Outlook on web on iPhone and Android phones.

Internal ADO item: 5776437

### CODE REVIEW

File: docs/concepts/requirements-for-running-office-add-ins.md [hunk 1]
```diff
@@ -63,22 +63,32 @@ The following are the minimum client versions of Office on Mac that support Offi
...
-|Outlook on the web|iPhone 4 or later, iPad 2 or later, iPod Touch 4 or later|iOS 5 or later|On Microsoft 365, Exchange Online, or on premises on Exchange Server 2013 or later|Safari|
+|Outlook on Android|- Android tablets<br>- Android smartphones|- Android 4.4 KitKat or later|On the latest update of Microsoft 365 Apps for business or Exchange Online|Browser not applicable. Use the native app for Android.<sup>1</sup>|
+|Outlook on iOS|- iPad tablets<br>- iPhone smartphones|- iOS 11 or later|On the latest update of Microsoft 365 Apps for business or Exchange Online|Browser not applicable. Use the native app for iOS.<sup>1</sup>|
+|Outlook on the web (modern)<sup>2</sup>|- iPad 2 or later<br>- Android tablets |- iOS 5 or later<br>- Android 4.4 KitKat or later|On Microsoft 365, Exchange Online|- Microsoft Edge<br>- Chrome<br>- Firefox<br>- Safari|
+|Outlook on the web (classic)|- iPhone 4 or later<br>- iPad 2 or later<br>- iPod Touch 4 or later|- iOS 5 or later|On on-premises Exchange Server 2013 or later|Safari|
```
Comment (ElizabethSamuel-MSFT) [inline docs/concepts/requirements-for-running-office-add-ins.md:75]:
```suggestion
|Outlook on the web (classic)|- iPhone 4 or later<br>- iPad 2 or later<br>- iPod Touch 4 or later|- iOS 5 or later|On on-premises Exchange Server 2013 or later|- Safari|
```

**Review by ElizabethSamuel-MSFT — APPROVED** [review]
Looks great! Just one small suggestion.

File: docs/concepts/requirements-for-running-office-add-ins.md [hunk 2]
```diff
@@ -63,22 +63,32 @@ The following are the minimum client versions of Office on Mac that support Offi
...
+
+> [!NOTE]
+> <sup>1</sup> OWA for Android, OWA for iPad, and OWA for iPhone native apps have been [deprecated](https://support.microsoft.com/office/076ec122-4576-4900-bc26-937f84d25a4b).
+>
+> <sup>2</sup> Modern Outlook on the web on iPhone and Android smartphones is no longer required or available for testing Outlook add-ins.
 
-> [!IMPORTANT]
-> The following are no longer required or available for testing Outlook add-ins.
+> [!TIP]
```
Comment (ElizabethSamuel-MSFT) [inline docs/concepts/requirements-for-running-office-add-ins.md:82]: Consider putting this tip into an include file then referencing that from here and the requirement sets page

File: docs/concepts/requirements-for-running-office-add-ins.md (hunk 1 above)
Comment (samantharamon) [reply docs/concepts/requirements-for-running-office-add-ins.md]: Thanks, Elizabeth! I was making the same change before I saw your comment, but thanks for suggesting it. :)

File: docs/concepts/requirements-for-running-office-add-ins.md (hunk 2 above)
Comment (samantharamon) [reply docs/concepts/requirements-for-running-office-add-ins.md]: Thanks! Created an include file for the tip block and referenced it here and in [docs/reference/requirement-sets/outlook-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/samramon-update-mobile-browser-support/docs/reference/requirement-sets/outlook-api-requirement-sets.md).

**Pull request merged**